spec/src/main/asciidoc/metrics_spec.adoc

```diff
@@ -184,6 +184,8 @@ The Metadata:
 * displayName (optional): A human readable name of the metric for display purposes if the metric name is not
 human readable. This could e.g. be the case when the metric name is a uuid.
 * tags (optional): A list of `key=value` pairs, which are separated by comma. See also <<supplying_of_tags>>.
+* reusable (optional): If set to `true`, then it is allowed to register a metric multiple times under the same name.
+Note that all such instances must set `reusable` to `true`. Default is `false`. See also <<reusing_of_metrics>>.
```

Comment:
We should say what implementers should do in case there is a mismatch in what reusable is set to.